CarCleaningService.rb

```diff
@@ -0,0 +1,145 @@
+class Car
+
+  attr_reader :start_time
+  attr_accessor :pickup_time, :car_time
+
+  def initialize()
+    @start_time = Time.now
+    #time when we start to wash your car
+    @car_time = nil
+    @pickup_time = nil
+  end
+end
+
+class Service 
+    OPEN_SERVICE = 8
+    CLOSE_SERVICE = 20
+    CAR_CLEANING_TIME = 2
+    def initialize()
+      @worker1 = Array.new
+      @worker2 = Array.new
+    end
+
+    def pickup(car)
+      #when both are free
```

Comment:
I think these comments are unnecessary. Also, you should try to space out the method a little bit, like adding new lines between different blocks of code, before ifs/loops

This applies to multiple methods, a short refactor just to space things out a little bit and remove all the comments would make a big difference